package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-edit-ui",
-  "version": "4.5.6",
+  "version": "4.5.6a",
```

Comment:
As this is a fix for what's in Prod right now (4.5.6), I chose this app version instead of 4.5.7. (This is also in a release branch, fwiw.)